content/applications/general/iot/connect.rst

```diff
@@ -0,0 +1,153 @@
+
+Click the card to access the list of :doc:`devices <devices>` connected to the IoT system and the
+IoT's system's :guilabel:`Technical Information`, such as its :guilabel:`Identifier`,
+:guilabel:`Domain address`, and :guilabel:`Image version`.
+
+Click :guilabel:`Download Logs` to download the IoT system's log file.
+
+.. tip::
+   To define the log levels recorded in the IoT system's log file, :ref:`access the IoT box's
+   <iot/windows-iot/homepage>` or :ref:`Windows virtual IoT's homepage <iot/iot-box/homepage>`,
+   click the :icon:`fa-cogs` (:guilabel:`cogs`) button, and click :guilabel:`Log level` at the
+   bottom of the page.
+
+.. note::
+   By default, drivers are automatically :ref:`udpated <iot_updating_iot/handlers>` every time the
+   IoT system is restarted. To disable automatic updates, uncheck the :guilabel:`Automatic drivers
+   update` option.
+
+.. _iot/connect/troubleshooting:
+
+Troubleshooting
+===============
+
+The pairing code does not appear or does not work
+-------------------------------------------------
+
+The :ref:`pairing code <iot/connect/pairing-code>` might not be displayed or printed under the
+following circumstances:
+
+- The IoT system is not connected to the Internet.
+- The IoT system is already connected to an Odoo database.
+- The :ref:`pairing code <iot/connect/pairing-code>` display time has expired. Reboot the IoT box
+  or :ref:`restart the Windows virtual IoT service <iot/windows_iot/restart>` to display the pairing
+  code again.
+- The IoT system's image version is too old and needs to be :ref:`updated
+  <iot/updating_iot/image-code>`.
+
+The IoT system is connected but does not appear in the database
+---------------------------------------------------------------
+
+The IoT system might take a few minutes to restart when it connects to a database. If it still does
+not appear after a few minutes:
+
+- Verify that the IoT system can reach the database and the server does not use a multi-database
+  environment.
+- Reboot the IoT box or :ref:`restart the Windows virtual IoT service <iot/windows_iot/restart>`.
+
+The IoT box is connected to the Odoo database but cannot be reached
+-------------------------------------------------------------------
+
+Verify that the IoT system and the computer running the Odoo database are connected to the same
+network.
+
+The Windows virtual IoT's homepage cannot be accessed from another device
+-------------------------------------------------------------------------
+
+Check the :ref:`iot/windows-iot/firewall`.
+
+The IoT system is disconnected from the database after an Odoo upgrade
+----------------------------------------------------------------------
+
+:ref:`Update the IoT system's image <iot/updating_iot/image-code>` by flashing the IoT box's card or
+:ref:`uninstalling the Windows virtual IoT program <iot/windows_iot/uninstall>` and
+:ref:`reinstalling <iot/windows-iot/installation>` the latest package.
```

Comment:
```suggestion
:ref:`reinstalling <iot/windows-iot/installation>` the latest release of the same version as your Odoo database.
```
! can be ambigious and migh thave customers installing version 18 iot box with 17 odoo database